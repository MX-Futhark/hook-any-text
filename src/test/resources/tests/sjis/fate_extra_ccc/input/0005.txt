97aa8fcc23524f46532d30303823525542538140835a83898374235255424523433030303234363230348740874187428743874423434445462352454e4482c991a282e782ea82bd89bc917a8bf38ad481410a23524f465330303032235255425382c282ad82dd82cd82e723525542458c8e8a438cb42352454e448a77898082c582b78142009b430000000000010000