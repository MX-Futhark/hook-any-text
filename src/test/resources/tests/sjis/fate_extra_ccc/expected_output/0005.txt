["略称ＳＥ．ＲＡ．ＰＨに造られた仮想空間、
月海原学園です。"]